cruise-control/src/main/scala/com/linkedin/kafka/cruisecontrol/executor/ExecutorUtils.scala

```diff
@@ -131,8 +131,4 @@ object ExecutorUtils {
   def getAllLiveBrokerIdsInCluster(kafkaZkClient: KafkaZkClient): java.util.List[java.lang.Integer] = {
     seqAsJavaList(kafkaZkClient.getAllBrokersInCluster.map(_.id : java.lang.Integer))
   }
-
-  def getAllTopicsInCluster(kafkaZkClient: KafkaZkClient): java.util.List[String] = {
```

Comment:
This method isn't called anywhere in the project so I removed it.

Reply:
Thanks for removing the dead code.